lib/asn1/base/node.js

```diff
@@ -1,7 +1,16 @@
-var assert = require('assert');
 var Reporter = require('../base').Reporter;
 var EncoderBuffer = require('../base').EncoderBuffer;
 
+function assert(val, msg) {
```

Comment:
This clearly does not belong here... What do you think about `minimalistic-assert` module? :) It seems that many of browserify modules may benefit from it.